src/Exception/MissingCommandException.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace League\Tactician\Container\Exception;
+
+use League\Tactician\Command;
+use League\Tactician\Exception\Exception;
+
+/**
+ * Command has not been added to the mapping array.
+ */
+class MissingCommandException extends \OutOfBoundsException implements Exception
```

Comment:
I'd recommend removing this custom exception and throw the `MissingHandlerException` in Tactician core instead. That would let folks always typehint for one particular exception, regardless of their background configuration. Also, there's a semantic error here in that we're missing a Handler, not a Command. :smile: